.github/workflows/ci.yaml

```diff
@@ -0,0 +1,135 @@
+name: ci
+
+on:
+  push:
+  pull_request:
+
+env:
+  BAZEL_CXXOPTS: -std=c++14
+
+jobs:
+  Linux-GCC:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Tests
+      run: >
+        bazel test ...
+        --copt=-DGTEST_REMOVE_LEGACY_TEST_CASEAPI_=1
+        --copt=-Werror
+        --cxxopt=-Wall
+        --cxxopt=-Wextra
+        --cxxopt=-Wcast-qual
+        --cxxopt=-Wconversion-null
+        --cxxopt=-Wformat-security
+        --cxxopt=-Wmissing-declarations
+        --cxxopt=-Wnon-virtual-dtor
+        --cxxopt=-Woverlength-strings
+        --cxxopt=-Wpointer-arith
+        --cxxopt=-Wundef
+        --cxxopt=-Wunused-local-typedefs
+        --cxxopt=-Wunused-result
+        --cxxopt=-Wvarargs
+        --cxxopt=-Wvla
+        --cxxopt=-Wwrite-strings
+        --cxxopt=-Wno-missing-declarations
+        --cxxopt=-Wno-undef
+        --features=external_include_paths
+        --keep_going
+        --show_timestamps
+        --test_env="TZDIR=${GITHUB_WORKSPACE}/testdata/zoneinfo"
+        --test_output=errors
+        --test_tag_filters=-benchmark
```

Comment:
If we have open source benchmarks, it would be good to at least build them.  
Do we still build them with this flag?

Reply:
Yes, we do build them.